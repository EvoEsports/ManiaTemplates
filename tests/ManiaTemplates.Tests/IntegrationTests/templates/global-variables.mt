<component>
    <property type="string" name="testVariable" />
    
    <template>
        <label text="{{ testVariable }}"/>
    </template>
</component>