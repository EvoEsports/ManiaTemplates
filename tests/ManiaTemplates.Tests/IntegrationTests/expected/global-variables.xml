<manialink version="3" id="MtGlobalVariables" name="EvoSC#-MtGlobalVariables">
<label text="unittest" />
</manialink>
